Tests/MarkdownTests/Parsing/BlockDirectiveParserTests.swift

```diff
@@ -1238,4 +1239,34 @@ class BlockDirectiveArgumentParserTests: XCTestCase {
         """#
         XCTAssertEqual(expected, document.debugDescription(options: .printSourceLocations))
     }
+    #endif
+}
+
+#if compiler(>=6.0) && canImport(Testing)
+import Testing
+
+struct _BlockDirectiveArgumentParserTests {
+    @Test(
+        "Directive MultiLine WithoutContent Parsing",
+        .bug("https://github.com/swiftlang/swift-markdown/issues" ,id: "#152", "Verify fix of #152")
+    )
+    func directiveMultiLineWithoutContentParsing() throws {
+        let source = #"""
+        @Image(
+          source: "example.png",
+          alt: "Example image"
+        )
+        """#
+        let document = Document(parsing: source, options: .parseBlockDirectives)
+        _ = try #require(document.child(at: 0) as? BlockDirective)
```

Comment:
```suggestion
        try #require(document.child(at: 0) as? BlockDirective != nil)
```

Reply:
BTW, what's the difference of `try #require(document.child(at: 0) as? BlockDirective != nil)` and `#expect(document.child(at: 0) as? BlockDirective != nil)`?

Reply:
`try #require()` will throw an error if it fails, stopping test execution. `#expect()` will keep going.